### PR TITLE
Update instructions for Kubernetes deployment of ODIM + URP, GRF and Dell plugins deployment

### PR DESCRIPTION
**Pull request by sunithasomanna**
Adding instructions for Kubernetes deployment of ODIMRA.
Adding instructions for URP, GRF and Dell plugins deployment + their addition to ODIM. 
Instructions are documented in the main readme, and the relevant plugin instructions are linked from the main readme to the readme present in the respective plugin dirs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,21 @@
-
-
 [![build_deploy_test Actions Status](https://github.com/ODIM-Project/ODIM/workflows/build_deploy_test/badge.svg)](https://github.com/ODIM-Project/ODIM/actions)
 [![build_unittest Actions Status](https://github.com/ODIM-Project/ODIM/workflows/build_unittest/badge.svg)](https://github.com/ODIM-Project/ODIM/actions)
 
 # Table of contents
 
+<<<<<<< Updated upstream
```
Comment (bharath-b-hpe) [inline README.md]: Please resolve the conflicts indicated by git

**Review by bharath-b-hpe — CHANGES_REQUESTED** [review]
Please sync the branch with upstream development, PR contains other changes too.

File: README.md (hunk 1 above)
Comment (sunithasomanna) [reply README.md]: Done! Resolved conflicts! Please verify

File: plugin-dell/README.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ This guide provides a set of guidelines for developing API and EMB functions to
 
 To ensure continued adoption of open technologies, the APIs for the plugins are based on the [OpenAPI specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md). Messaging is based on the now-evolving [OpenMessaging specifications](https://github.com/openmessaging/specification/blob/master/domain_architecture.md) under Linux Foundation.
 
+## Dell plugin deployment instructions
 
+For deploying the Dell plugin and adding the plugin to the Resource Aggregator for ODIM framework, refer to the "Deploying the Dell plugin" section in the [Resource Aggregator for Open Distributed Infrastructure Management™ Readme](https://github.com/ODIM-Project/ODIM/tree/development#readme).
 
 
 ## API accessibility
```
Comment (sukhwinder20) [inline plugin-dell/README.md]: This is not required in Dell plugin

**Comment by sunithasomanna** [comment]
Changing Title to include Kubernetes deployment of ODIMRA

**Comment by sunithasomanna** [comment]
Closing this PR because it is no longer valid. Had files other than the ones relevant to documentation. Raised a new PR for this and got the content approved and merged to upstream dev branch. Hence closing this PR.

**Pull request closed without merging**